chirp/drivers/tdh8.py

```diff
@@ -2144,6 +2091,15 @@ class TDH8_GMRS(TDH8):
     _vhf_range = (136000000, 175000000)
     _uhf_range = (400000000, 521000000)
 
+    def validate_memory(self, mem):
+        msgs = super().validate_memory(mem)
+        if mem.freq / 1000000 in GMRS_FREQS3:
```

Comment:
If you use the GMRS definitions in `bandplan_na` you won't have to divide here.

Reply:
Thanks, these values I can find in `bandplan_na`.